components/gfx/text/shaping/harfbuzz.rs

```diff
@@ -50,6 +50,8 @@ use std::ptr;
 static NO_GLYPH: i32 = -1;
 static CONTINUATION_BYTE: i32 = -2;
 
+static KERN: u32 = ((b'k' as u32) << 24) | ((b'e' as u32) << 16) | ((b'r' as u32) << 8) |
+    (b'n' as u32);
```

Comment:
Maybe a good intro bug for someone would be to write a syntax extension for this.  :)